File: web-app/client/src/app/user-task.ts
export class UserTask {
    id: number;

    constructor(
        public userId: number,
        public action: string,
        public dueDate: Date,
        public severity: UserTaskSeverity,
        public done: boolean
    ) { }
}

export enum UserTaskSeverity {
    LOW,
    NORMAL,
    HIGH
}

File: domain/UserTask.cs
using System;

namespace My.ToDoApp.Domain {
    public class UserTask: IEntity {
        public int Id { get; set; }
        public string Action { get; set; }
        public DateTime DueDate { get; set; }
        public UserTaskSeverity Severity { get; set; }
        public bool Done { get; set; }
        public User User { get; set; }
    }
}